images/win/toolcache-2019.json

```diff
@@ -0,0 +1,20 @@
+{
+    "@actions/toolcache-python-windows-x64": [
+        "2.7", "3.5", "3.6", "3.7", "3.8"
+    ],
+    "@actions/toolcache-python-windows-x86": [
+        "2.7", "3.5", "3.6", "3.7", "3.8"
+    ],
+    "@actions/toolcache-ruby-windows-x64": [
+        "2.4", "2.5", "2.6", "2.7"
+    ],
+    "@actions/toolcache-pypy-windows-x86": [
+        "2", "3"
+    ],
+    "@actions/toolcache-boost-windows-msvc-14.1-x32-x64": [
```

Comment:
same here, it seems we don't need boost 1.69

Reply:
We can't build boos 1.69 on VS19, but it still works. We can't delete it because it will be a regression.